lib/linter.sh

```diff
@@ -779,6 +787,16 @@ done
 GetStandardRules "javascript"
 GetStandardRules "typescript"
 
+#################################
+# Define extra opts for linters #
+#################################
+if IsLintly; then
```

Comment:
I have a slight feeling this lintly opts stuff isn't supposed to go here, but I'm not quite sure how else to do it (and still get `LINTER_COMMANDS_ARRAY` updated with the extra CLI options).